examples/nlp/gpt/train_gpt_sft.py

```diff
@@ -119,15 +126,18 @@ def main(cfg) -> None:
     with open_dict(cfg):
         cfg.model.precision = cfg.trainer.precision
 
-    ptl_model = load_from_nemo(
+    ptl_model, updated_cfg = load_from_nemo(
         GPTSFTModel,
         cfg,
         trainer,
         strict=True,
         modify_config_fn=_modify_config,
         restore_path=cfg.model.restore_from_path,
+        return_updated_cfg=True,
     )
 
+    init_peft(ptl_model, updated_cfg)
```

Comment:
we should just give the peft cfg

Reply:
Unfortunately this is not only peft cfg, it also contains base model cfg, because we need those (e.g num_attention_heads, hidden_size, num_layers) to initialize lora modules in [nemo](https://github.com/NVIDIA/NeMo/blob/5fdd12e9a9711b241023eb4ed0922733d69ded5e/nemo/collections/nlp/parts/peft_config.py#L69).

Reply:
ah okay. then i'm good with this.